dev-lang/idris/idris-1.3.3.ebuild

```diff
@@ -68,6 +68,9 @@ src_prepare() {
 	# runs dist/build/idris directly and breaks sandboxing
 	export LD_LIBRARY_PATH="${S}/dist/build${LD_LIBRARY_PATH+:}${LD_LIBRARY_PATH}"
 
+	# Upstream did this in commit cae9aa29
+	cabal_chdeps 'Cabal >= 2.4 && < 3.1' 'Cabal >= 2.4 && <= 3.4'
```

Comment:
`<=N` style depend are almost never correct: then allow `N`, but not `N.0.0.1`. I suggest dropping upper bound entirely.